core-bundle/contao/themes/flexible/styles/basic.css

```diff
@@ -927,28 +946,66 @@ a.tl_submit {
 	font-weight: 600;
 }
 
-@media (-webkit-min-device-pixel-ratio: 2), (min-resolution: 192dpi) {
-	.badge-title {
-		font-weight: 500;
+/* Tablets */
+@media (min-width: 768px) {
+	.widget-group {
+		--widget-gap-cols: 30px;
+	}
+
+	.w16, .w25, .w33, .w50, .w75 {
+		--widget-cols: 6;
+	}
+
+	/* Backwards compatibility */
+	*:not(.widget-group) > .w50 {
+		width: calc(50% - 30px);
+		float: left;
+		min-height: 80px;
+	}
+
+	.m12 {
+		margin: 0 15px;
+		padding: 18px 0 16px;
+	}
+
+	.cbx {
+		min-height: 46px;
+	}
+
+	.cbx.m12 {
+		min-height: 80px;
```

Comment:
This adds the minimum height again. Shouldn‘t it be `*:not(.widget-group) > .cbx.m12` instead?

Reply:
Updated these in the latest commit